pkg/discovery/processor/turbo_policy_processor.go

```diff
@@ -46,7 +46,8 @@ func (p *TurboPolicyProcessor) ProcessTurboPolicies() {
 	glog.V(2).Infof("Discovered %v PolicyBindings.", len(turboPolicyBindings))
 
 	policyMap := make(map[string]*repository.TurboPolicy)
-	for _, turboSloScaling := range turboSloScalings {
+	for _, sloScale := range turboSloScalings {
+		turboSloScaling := sloScale
```

Comment:
Can you elaborate why this makes a difference?  Is that because `:=` actually creates a copy?  If so, then would it make sense to perhaps name it `turboSloScalingCopy` to make it obvious?

Reply:
This is a great suggestion. Thanks @chlam4 .